File: 민균/2주차/code_2609.java
package baekjoon.study.week2;
import java.util.Scanner;

public class code_2609 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        long num1 = sc.nextLong();
        long num2 = sc.nextLong();

        long gcd = getGCD(Math.max(num1, num2), Math.min(num1, num2));
        long lcm = getLCM(num1, num2, gcd);

        System.out.println(gcd);
        System.out.println(lcm);
        sc.close();
    }

    public static long getGCD(long a, long b) {
        while(b > 0) {
            long tmp = a;
            a = b;
            b = tmp%b;
        }
        return a;
    }

    public static long getLCM(long a, long b, long gcd) {
        return (a*b)/gcd;
    }
}

File: 혜원/week2/code_2609.cpp
//2609

#include <iostream>
using namespace std;

int gcd(int a, int b) {
	while (b != 0) {
		int r = a % b;
		a = b;
		b = r;
	}
	return a;
}
int lcm(int a, int b) {
	return a * b / gcd(a, b);
}

int main() {
	int a = 0, b = 0, min = 0, max = 0;

	cin >> a >> b;
	if (a > 10000 || b > 10000) return 0;

	max = gcd(a, b);
	min = lcm(a, b);

	cout << max << '\n' << min << endl;

	return 0;
}

////유클리드 호제법